src/components/Table/Table.jsx

```diff
@@ -268,6 +269,8 @@ export const defaultProps = baseProps => ({
     batchCancel: 'Cancel',
     itemsSelected: 'items selected',
     itemSelected: 'item selected',
+    rowCountLabel: 'Results',
```

Comment:
I would just leave the rowCountLabel as a function (which is what you have previous).  My discussion with taylor was more about how we should find a better way for all component to manage this, but, using a function for the label is what we have today, and as such that should be used.  Right now, you've just turned 1 label prop into 3 which isn't better :(

Reply:
rowCountInHeader func calling moved in Toolbar.jsx and removed rowCountHeaderField. Please review again.

Reply:
I think we are still missing something here.   Just get rid of rowCountLabel since on it's own, it is not used.  You are later passing down into another function and then telling that function to compose some items plus a base label, and you should never do that from a translation point of view.  All I'm saying is that, remove all reference rowCountLabel then then in your rowCountInHeader function label, just have it be defined like...
```
rowCountInHeader: (totalRowCount, rowCountLabel) => `Results: ${totalRowCount}`,
```

The reason why you should not pass a label into a translation function is what if I wanted to display `247 Items Returned` instead of `Results: 247`.  The way you have this setup, it would basically pass `Items Returned` to the format function and then we'd have `Items Returned: 247`.

In essent, just treat your label function as if it were a static piece of text where you'll insert some number.  No need to break it up into smaller parts and then try to comose it, since, translation will fail miserable on that kind of composition of labels.

Reply:
@stuckless suggested changes are added in latest commit. please review.